### PR TITLE
Added expiration to jwt created by frontend

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

Added expiration to the jet created by the frontend, and check for jwt expiration on all pages. Updated jwt expiration checks to redirect to new error page.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested locally to ensure jwt expiration had the desired effect.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: SPDBCSC-274

### CODE REVIEW

File: src/ecrc-frontend/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "ecrc-frontend",
   "version": "0.1.0",
+  "homepage": "/criminalrecordcheck",
```
Comment (sdevalapurkar) [inline src/ecrc-frontend/package.json]: does this work? 😮

Comment (adodge-bcg) [reply src/ecrc-frontend/package.json]: No, I think this must have been left in before I switched branches =( It doesn't work, and I broke everything

**Comment by adodge-bcg** [comment]
> can we make it so the existing route protection also redirects to error page with a message like "please follow sequential flow" or something instead of going to home page? :)

We can, but I think that should be another ticket as that is creating a new view of the error page. Unless you think that should be part of the jwt timeout

File: src/ecrc-frontend/package.json (hunk 1 above)
Comment (adodge-bcg) [reply src/ecrc-frontend/package.json]: Removed

**Pull request merged**